### PR TITLE
Add Foundry VTT subdomain

### PR DESCRIPTION
**Pull request by Sakujakira**
Added capabillity for hosting [Foundry VTT](https://foundryvtt.com/).

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]
	
Aiming to resolve Issue #303

### CODE REVIEW

File: foundryvtt.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        #auth_request /auth;
+        #error_page 401 =200 /ldaplogin;
+
+        # enable for Authelia
+        #include /config/nginx/authelia-location.conf;
+
+        include /config/nginx/proxy.conf;
+        resolver 127.0.0.11 valid=30s;
+        set $upstream_app <container-name>;
+        set $upstream_port <port>;
```
Comment (Roxedus) [inline foundryvtt.subdomain.conf.sample:35]: Please fill these

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: foundryvtt.subdomain.conf.sample (hunk 1 above)
Comment (Sakujakira) [reply foundryvtt.subdomain.conf.sample]: I`ve changed these two lines and updated this merge request

File: foundryvtt.subdomain.conf.sample [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        # enable the next two lines for ldap auth
+        #auth_request /auth;
+        #error_page 401 =200 /ldaplogin;
+
+        # enable for Authelia
+        #include /config/nginx/authelia-location.conf;
+
+        include /config/nginx/proxy.conf;
+        resolver 127.0.0.11 valid=30s;
```
Comment (Roxedus) [inline foundryvtt.subdomain.conf.sample:33]: Please update this to reflect the changes brought by #330

File: foundryvtt.subdomain.conf.sample [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        set $upstream_app foundry;
+        set $upstream_port 30000;
+        set $upstream_proto http;
+        proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+
+        # Set proxy headers
+        proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
```
Comment (Roxedus) [inline foundryvtt.subdomain.conf.sample:42]: These are covered in the included proxy.conf.
https://github.com/linuxserver/docker-swag/blob/master/root/defaults/proxy.conf#L26-L28

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Pull request closed without merging**